src/main/java/io/woo/htmltopdf/HtmlToPdf.java

```diff
@@ -12,9 +8,12 @@
 import java.util.Map;
 import java.util.function.BiFunction;
 import java.util.function.Consumer;
-import java.util.function.Function;
 import java.util.stream.Collectors;
 
+import com.sun.jna.Pointer;
+import com.sun.jna.ptr.PointerByReference;
+import io.woo.htmltopdf.wkhtmltopdf.WkHtmlToPdf;
```

Comment:
Unused imports

Reply:
I do not see any unused imports. These imports were present before as well. It is just reorganized by the IDE to have the `java.*` classes first and others later.

Reply:
Thanks 👍 😃